### PR TITLE
feat(AppGallery): add presence

### LINKED ISSUE #5380: AppGallery | appgallery.huawei.com
### Discussed in https://github.com/PreMiD/Presences/discussions/4559

<div type='discussions-op-text'>

<sup>Originally posted by **MaXfAa** February 21, 2021</sup>
**Prerequisites and essential questions** 
- [x] Is it a popular site?
- [x] Is the website older than 2 months? 
- [ ] Is the site locked to a specific country/region?
- [ ] Is the site a paid service? (e.g. Netflix, Hulu)
- [ ] Does the website feature NSFW content? (e.g. porn, etc...)
- [ ] Are you a donator/patron?
- [x] Do you acknowledge that coding presences is completely voluntary and may take time for your service to be added regardless of priority?

**What's your Discord username?** 
Maxou#1909

**What's the name of the service?** 
https://appgallery.huawei.com | (Huawei) AppGallery

**What should the Presence display?** 
Same that Google Play

**If possible, please provide a logo for the service (512x512 minimum)** 

https://appimg-drru.dbankcdn.com/application/icon144/10167/a47479a364a34535b4819f2fb2e80a3e.png</div>

### PR DESCRIPTION
**Pull request by AngeloCore**
**Describe the changes in this pull request:**

<details>
<summary> Proof to proving the creation/modification working </summary>
<br>

</details>

Resolves #5380

### CODE REVIEW

**Comment by AngeloCore** [comment]
Resolves #5380

File: websites/A/App Gallery/presence.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    const pdURL = await (
+      await fetch(`https://pd.premid.app/create/${url}`)
+    ).text();
+    shortenedURLs[url] = pdURL;
+    return pdURL;
+  } catch (err) {
+    presence.error(err);
+    return url;
+  }
+}
```
Comment (EncryptedDev) [inline websites/A/App Gallery/presence.ts:72]: From my tests, nothing exceeds Discord's ratelimit, is this actually needed?

Comment (AngeloCore) [reply websites/A/App Gallery/presence.ts:72]: no

**Comment by AngeloCore** [comment]
.

**Comment by AngeloCore** [comment]
So this will get closed? xD

**Comment by EncryptedDev** [comment]
Low quality pull request

**Pull request closed without merging**